docs/pages/ESSupport.md

```diff
@@ -0,0 +1,368 @@
+
+## ECMAScript Version Support
+
+The UI5 Tooling offers general support for the latest ECMAScript features. While a `ui5 build` is executed, the UI5 Tooling analyses the code in a few places. Depending on your project you have to consider some restrictions regarding the used ECMAScript syntax.
+
+## General
+The following sections describe the limitations valid for all types of projects.
+
+### sap.ui.define/sap.ui.require: Usage of Object Expressions, Spread Elements, and Template Literals
+An `Object Expression`, `Spread Element`, or `Template Literal` with an expression is not supported as a parameter for `sap.ui.define` or `sap.ui.require`.
```

Comment:
put a list which other APIs are similar handled:
- `jQuery.sap.declare`
- `jQuery.sap.require`
.....

Reply:
add mark the deprecated ones as deprecated